website/source/api/agent/service.html.md

```diff
@@ -151,6 +151,194 @@ contains the [hash-based blocking
 query](/api/index.html#hash-based-blocking-queries) hash for the result. The
 same hash is also present in `X-Consul-ContentHash`.
 
+## Get local service health
+
+The `/v1/agent/health/service/name/<service_name>` and
+`/v1/agent/health/service/id/<service_id>` endpoints allow retrieving an
+aggregated state of service(s) of the local agent.
+
+The endpoint `/v1/agent/health/service/name/<service_name>` queries all
+services with a given names (several may match), while
+`/v1/agent/health/service/id/<service_id>` will match a single service only.
+
+If you know the ID of service you want to target, it is recommended to use
+the version `/v1/agent/health/service/id/<service_id>` so you have the result
+for the service only. When requesting
+`/v1/agent/health/service/name/<service_name>`, the caller will receive the
+worst state of all services having the given name.
+
+Those endpoints support JSON format and text/plain formats, JSON being the
+default. In order to get the text format, you can append `?format=text` to
+the URL or use Mime Content negotiation by specifying a HTTP Header
+`Accept` starting with `text/plain`.
+
+Those endpoints return the aggregated values of all healthchecks for the
+service and will return the corresponding HTTP codes:
+
+| Result | Meaning                                                |
+| ------ | ------------------------------------------------------ |
+| `200`  | All healthchecks of this service are passing           |
+| `400`  | Bad parameter (missing service name of id)             |
+| `404`  | No such service id or name                             |
+| `429`  | Some healthchecks are passing, at least one is warning |
+| `503`  | At least one of the healthchecks is critical           |
+
+Those endpoints might be usefull for the following use-cases:
+
+* a load-balancer wants to check IP connectivity with an agent and retrieve
+  the aggregated status of given service
+* create aliases for a given service (thus, the healthcheck of alias uses
+  http://localhost:8500/v1/agent/service/id/aliased_service_id healthcheck)
+
```

Comment:
These docs and examples are really great thanks!

Can we split the endpoints to make the consistent with all the other endpoint docs though please?

For example if we have other endpoints like `/catalog/service/:name` and `/catalog/connect/:name` that are almost identical except for one semantic detail we just document one and then make a stub with links in the other etc: https://www.consul.io/api/catalog.html#list-nodes-for-connect-capable-service

I think that patter would work fine here although the response format is subtly different so maybe only the description park would be shared/linked between.

Once, split, can we follow the same format as all other endpoints. i.e.:

```
## Title

Description of purpose

[Extended notes]

| Method | Path                         | Produces                   |
| ------ | ---------------------------- | -------------------------- |
| `GET`  | `/path`            | `application/json`         |

The table below shows this endpoint's support for
[blocking queries](/api/index.html#blocking-queries),
[consistency modes](/api/index.html#consistency-modes),
[agent caching](/api/index.html#agent-caching), and
[required ACLs](/api/index.html#acls).

| Blocking Queries | Consistency Modes | Agent Caching | ACL Required   |
| ---------------- | ----------------- | ------------- | -------------- |
| `NO`             | `none`            | `none`        | `service:read` |

### Sample Request

```text
$ curl \
    http://127.0.0.1:8500/v1/agent/services
```

### Sample Response

```json
{
  "redis": {
      "ID": "redis",
      "Service": "redis",
      "Tags": [],
      "Meta": {
          "redis_version": "4.0"
      },
      "Port": 8000,
      "Address": "",
      "EnableTagOverride": false,
      "Weights": {
          "Passing": 10,
          "Warning": 1
      }
  }
}
```

I suggest the format variants you make the table like we did for metrics (in this same file!):

![image](https://user-images.githubusercontent.com/120915/48262076-199f8480-e419-11e8-91cf-fca4b38bf983.png)

Thanks!